nautobot/ipam/api/serializers.py

```diff
@@ -135,6 +136,37 @@ def validate(self, data):
         return data
 
 
+class VLANLegacySerializer(VLANSerializer):
```

Comment:
Mind adding a docstring spelling out that this is used for API versions 2.0 and 2.1?